### PR TITLE
fix(profiling): fix bad query and update wording

### PR DESCRIPTION
**Pull request by JonasBa**

### CODE REVIEW

File: static/app/utils/profiling/hooks/useAggregateFlamegraphQuery.ts [hunk 1]
```diff
@@ -20,6 +20,9 @@ export function useAggregateFlamegraphQuery({transaction}: {transaction: string}
     return conditions.formatString();
   }, [transaction]);
 
+  const enabled =
+    !!transaction && Array.isArray(selection.projects) && selection.projects.length > 0;
```
Comment (Zylphrex) [inline static/app/utils/profiling/hooks/useAggregateFlamegraphQuery.ts:24]: Is this to not fire the query before the selection is ready? This does assume that the flamegraph query cant be cross-project which is probably a reasonable assumption.

File: static/app/components/profiling/profileEventsTable.tsx [hunk 2]
```diff
@@ -155,31 +155,28 @@ function ProfileEventsCell<F extends FieldType>(props: ProfileEventsCellProps<F>
...
-    const traceId = props.dataRow[key];
+    const traceId = getShortEventId(props.dataRow[key] ?? '');
     if (!traceId) {
       return <Container>{t('n/a')}</Container>;
     }
 
     return (
       <Container>
-        <Link to={`/performance/trace/${getShortEventId(traceId)}`}>{traceId}</Link>
+        <Link to={`/performance/trace/${traceId}`}>{traceId}</Link>
```
Comment (Zylphrex) [inline static/app/components/profiling/profileEventsTable.tsx:165]: The link should use the full trace id

File: static/app/components/profiling/profileEventsTable.tsx [hunk 3]
```diff
@@ -155,31 +155,28 @@ function ProfileEventsCell<F extends FieldType>(props: ProfileEventsCellProps<F>
...
     if (!project) {
       return <Container>{transactionId}</Container>;
     }
 
     return (
       <Container>
-        <Link to={`/performance/${project.slug}:${getShortEventId(transactionId)}`}>
-          {transactionId}
-        </Link>
+        <Link to={`/performance/${project.slug}:${transactionId}`}>{transactionId}</Link>
```
Comment (Zylphrex) [inline static/app/components/profiling/profileEventsTable.tsx:179]: The link should use the full transaction id

**Review by Zylphrex — APPROVED** [review]
A few minor comments but otherwise LGTM

File: static/app/utils/profiling/hooks/useAggregateFlamegraphQuery.ts (hunk 1 above)
Comment (JonasBa) [reply static/app/utils/profiling/hooks/useAggregateFlamegraphQuery.ts:24]: yeah, correct. I think we always end up with a single project here, so I think its fine too

**Pull request merged**